support/azure/azure-kubernetes/availability-performance/node-auto-repair-errors.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Troubleshoot common node auto-repair errors
+description: Troubleshoot scenarios where node auto-repair returns an error code when trying to repair your NotReady node.
+ms.date: 10/01/2024
+ms.reviewer: 
+ms.service: azure-kubernetes-service
+#Customer intent: As an Azure Kubernetes user, I want to make sure the automatic repair actions from AKS node auto-repair do not cause any impacts on my applications or cluster health.
+ms.custom: sap:Node/node pool availability and performance
+---
+# Troubleshoot common node auto-repair errors
+
+When AKS detects a node which reports the NotReady status for more than 5 minutes, we will attempt to automatically repair your node. Node auto-repair is a best-effort service and does not guarantee that your node will be restored to a healthy state. Learn more about the [node auto-repair process](/azure/aks/node-auto-repair).
```

Comment:
How do you suggest to word this without saying "AKS" twice in the same sentence? Right now it will be: "When AKS detects a node which reports the NotReady status for more than 5 minutes, AKS will attempt to automatically repair your node."